apps/avatax/src/modules/utils/assert-unreachable.ts

```diff
@@ -0,0 +1,3 @@
+export function assertUnreachableButNotThrow(_: never) {
+  return null as never;
+}
```

Comment:
not sure if we need hacks like this. we can throw unhandled error in the end

Reply:
Fixed - changed to throw instead

Reply:
Actually I decided not go with throwing route - it was causing ts to shout (with the reason) so I created a noop `assertUnreachableWithoutThrow` - it will assert `never` but return provided value instead of `null`